requirements/pip.txt

```diff
@@ -15,7 +15,7 @@ six==1.11.0
 future==0.16.0
 
 # django-tastypie 0.13.x and 0.14.0 are not compatible with our code
```

Comment:
Let's update this comment. From the [tastypie changelog](https://django-tastypie.readthedocs.io/en/latest/release_notes/v0.13.3.html), it looks like 0.13.3 is still compatible with Django 1.9.